restconnector/RestConnector.h

```diff
@@ -11,20 +11,35 @@
 namespace restconnector
 {
 
+/**
+ * The RestConnector class is responsible starting a HTTP server
+ * in a dedicated thread, according to a given hostname and port.
+ * HTTP requests are forwarded to an implicitely registered handler.
+ */
 class RestConnector
 {
 public:
 
-    RestConnector( const std::string& address, const std::string& port );
+    /**
+     * Default constructor.
+     * @param hostname Hostname or IP address
+     * @param port Listening port
+     */
+    RestConnector( const std::string& hostname, const std::string& port );
```

Comment:
I definitely agree this should be tested. That's in the list of things that are already planned. The only problem I have with changing the interface is that the user of the RESTConnector class becomes responsible for checking that the port (that will usualy come from as a string from the command line) is actually a number. I would rather leave this check to the RESTConnector and raise a proper exception when the port (or any other parameter) is invalid. Still I agree that the port should be a number ;-)